.github/workflows/CI-unittests.yml

```diff
@@ -26,25 +26,25 @@ jobs:
             python-version: "3.9"
             install-method: mamba
 
-          - os: ubuntu-latest
-            python-version: "3.9"
-            install-method: pip
+#          - os: ubuntu-latest
```

Comment:
We usually don't comment out code (bad practice). I suggest we remove the comments

Reply:
Yes I know. My suggestion is to keep it, as this allows us to switch it on much easier again.